WooCommerce/Classes/ViewRelated/Orders/Order Details/Address Edit/FilterListSelector.swift

```diff
@@ -1,21 +1,42 @@
 import SwiftUI
 
-/// Country Selector View
+protocol FilterListSelectorViewModelable: ObservableObject {
```

Comment:
I named it like that because of this rule.

> Protocols that describe a capability should be named using the suffixes able, ible, or ing (e.g. Equatable, ProgressReporting).

[Source](https://swift.org/documentation/api-design-guidelines/)

Reply:
I always struggle with those suffixes! 😅